scan/scan.go

```diff
@@ -45,6 +45,13 @@ func NewScanner(opts options.Options, cfg config.Config) (Scanner, error) {
 		return nil, err
 	}
 	if st == typeDirScanner {
+		if opts.AdditionalConfig != "" && err == nil {
```

Comment:
Err check is not necessary
```suggestion
		if opts.AdditionalConfig != ""  {
```

Reply:
Removed